lib/mayaUsd/nodes/proxyShapeListenerBase.cpp

```diff
@@ -0,0 +1,227 @@
+//
+// Copyright 2023 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#include "proxyShapeListenerBase.h"
+
+#include <pxr/usd/usdUtils/stageCache.h>
+
+#include <maya/MDataBlock.h>
+#include <maya/MDataHandle.h>
+#include <maya/MFnNumericAttribute.h>
+
+#include <mutex>
+
+using namespace MAYAUSD_NS_DEF;
```

Comment:
The macro is only for declaring the namespace so that it gets versioned.. When using the namespace, just use the name "MayaUsd" which will never change.